docs/src/manual/models.md

```diff
@@ -74,6 +74,44 @@ CachingOptimizer state: EMPTY_OPTIMIZER
 Solver name: GLPK
 ```
 
+## Print the model
+
+By default, `show(model)` will print a summary of the problem.
+```jldoctest model_print
+julia> model = Model(); @variable(model, x >= 0); @objective(model, Max, x);
+
+julia> model
+A JuMP Model
+Maximization problem with:
+Variable: 1
+Objective function type: VariableRef
+`VariableRef`-in-`MathOptInterface.GreaterThan{Float64}`: 1 constraint
+Model mode: AUTOMATIC
+CachingOptimizer state: NO_OPTIMIZER
+Solver name: No optimizer attached.
+Names registered in the model: x
+```
+
+Use [`print(::AbstractModel)`](@ref) to print the formulation of the model.
+```jldoctest model_print
+julia> print(model)
+Max x
+Subject to
+ x ≥ 0.0
+```
+!!! warning
+    This format is specific to JuMP. To write the model to a file, use
+    [`write_to_file`](@ref) instead.
+
+
+Pass `latex = true` to print the model in LaTeX form.
+```jldoctesst model_print
```

Comment:
tesst -> test

Reply:
Fixed. You would think this should throw an error...